client/src/Controller/AbstractController.php

```diff
@@ -14,7 +14,7 @@ abstract class AbstractController extends Controller
      */
     protected function renderError(string $description, $statusCode = 500)
     {
-        $text = $this->renderView('TwigBundle/Exception/template.html.twig', [
+        $text = $this->renderView('bundles/TwigBundle/Exception/template.html.twig', [
```

Comment:
Oh snap was this broken the whole time? Sure I had tested this locally when I made the change 😭